### PR TITLE
feat(routific) build fleet from calendars

### PR DESCRIPTION
**Pull request by gabiseabra**

### CODE REVIEW

File: apps/re_integrations/lib/routific/payload/outbound.ex [hunk 1]
```diff
@@ -45,11 +46,39 @@ defmodule ReIntegrations.Routific.Payload.Outbound do
 
   defp build_visit(_visit), do: :error
 
-  defp build_fleet(_visits) do
-    # TODO build fleet from photographers' google calendars
-    {:ok, %{}}
+  defp build_fleet(visits) do
+    {:ok, visits
```
Comment (rhnonose) [inline apps/re_integrations/lib/routific/payload/outbound.ex]: If you're expecting no errors here, the `{:ok, result}` tuple is unnecessary.
Maybe you should add a `case` statement at the end when `get_calendars/1` return an empty list.

File: apps/re_integrations/lib/routific/payload/outbound.ex [hunk 2]
```diff
@@ -45,11 +46,39 @@ defmodule ReIntegrations.Routific.Payload.Outbound do
...
+  end
+
+  defp get_calendars(neighborhoods) do
+    Calendar
+    |> Calendar.Queries.by_district_names(neighborhoods)
+    |> Calendar.Queries.preload_relations([:address])
+    |> Re.Repo.all()
+  end
+
+  defp build_depot(%{address: address}), do: %{
```
Comment (rhnonose) [inline apps/re_integrations/lib/routific/payload/outbound.ex]: Did you mean `depart`? I'm not sure what `depot` would mean in this context, so just guessing 😄

Comment (gabiseabra) [reply apps/re_integrations/lib/routific/payload/outbound.ex]: This is routific's terminology for a vehicle's starting point.

File: apps/re_integrations/lib/routific/payload/outbound.ex [hunk 3]
```diff
@@ -45,11 +46,40 @@ defmodule ReIntegrations.Routific.Payload.Outbound do
 
   defp build_visit(_visit), do: :error
 
-  defp build_fleet(_visits) do
-    # TODO build fleet from photographers' google calendars
-    {:ok, %{}}
+  defp build_fleet(visits) do
+    with {:ok, calendars} <- visits |> get_neighborhoods() |> get_calendars() do
```
Comment (pmargreff) [inline apps/re_integrations/lib/routific/payload/outbound.ex]: You're not threating errors on `with`, use it without `with` as 
```
{:ok, _} = a 
|> foo()
|> bar()
|> baz()
```
could be good enough.

Comment (rhnonose) [reply apps/re_integrations/lib/routific/payload/outbound.ex]: Or just flat remove `{:ok, _} = a`

**Pull request merged**